### PR TITLE
Add initial files of project

### PR DESCRIPTION
**Pull request by rvente**
image

- Grid up and running
- clicking cell updates color
- cells are square, as desired

### CODE REVIEW

File: js/setColor.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// add the `clicked` class to changed the color
```
Comment (rvente) [inline js/setColor.js:1]: This comment is out of date.

File: js/constants.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const initial = {
```
Comment (rvente) [inline js/constants.js:1]: Should we use SCREAMING_SNAKE_CASE for these constant global variables?

File: css/grid-app.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.cell {
+  width: 10px; /* cell sizes */
+  height: 10px;
+  padding: 0px; /* pad prevents cells from being square */
+  border: 1px solid white; /* add contrast between individual cells */
+}
+
+/* temporary classes to set up non-default inital color */
+/* to be overridden by javascript which sets node color from initial state */
```
Comment (rvente) [inline css/grid-app.css:9]: The js color override comment may be deleted. - I've already done that.

File: js/constants.js (hunk 2 above)
Comment (xuejingao) [reply js/constants.js:1]: I'm not familiar with the best practices; but we could use it if we stay consistent with it?

**Review by xuejingao — APPROVED** [review]
lgtm; I tried to load the javascript files dynamically but I ran into loading issues since the html couldn't get access to variables because it rendered first.

**Pull request merged**